src/graphql/TeamSettings/flows.gql

```diff
@@ -4,5 +4,8 @@ query Flows {
     archived
     name
     version_group_id
+    project {
```

Comment:
Could you add `id` in the project join here? Vue-Apollo doesn't do well without that field and can return really weird results in some caching scenarios.